vite-project/src/components/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { IcArrow, IcHome } from '../assets/svgs/icons';
+
+const Header = () => {
+    return (
+        <Head>
+            <IcArrow />
+            <HeadTextWrapper>
+                장바구니 <HeadTextBox>(2)</HeadTextBox>
+            </HeadTextWrapper>
+            <IcHome />
+        </Head>
+    );
+};
+
+export default Header;
+
+const Head = styled.div`
```

Comment:
시맨틱하게 header 써보는건 어때?
```suggestion
const Head = styled.header`
```

Reply:
바로 반영했습니다!